### DISCUSSION TITLE
Looking for a list of delivered icon names

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#2600] [Q&A] [answered]
The the third argument of  the function ToolbarItemUtilities.createActionButton() takes an icon name.  For example, "icon-circle".
Is there a list of all the delivered icon names?

**Comment by aruniverse** [comment, marked as answer]
Take a look at: https://unpkg.com/@bentley/icons-generic-webfont@latest/dist/bentley-icons-generic-webfont.html

**Comment by JohnMajerle** [comment]
Thanks!  That's exactly what I was looking for.
